packaging/installer/README.md

```diff
@@ -6,150 +6,8 @@ Netdata is very flexible and can be used to monitor all kinds of infrastructure.
-those who run Netdata in _mission-critical production systems_, as updates will come more infrequently, and only after
-the community helps fix any bugs that might have been introduced in previous releases.
-
-**Pros of using nightly releases:**
-
-- Get the latest features and bug fixes as soon as they're available
-- Receive security-related fixes immediately
-- Use stable, fully-tested code that's always improving
-- Leverage the same Netdata experience our community is using
-
-**Pros of using stable releases:**
-
-- Protect yourself from the rare instance when major bugs slip through our testing and negatively affect a Netdata installation
-- Retain more control over the Netdata version you use
-
-### Anonymous statistics
-
-Starting with v1.30, Netdata collects anonymous usage information by default and sends it to a self-hosted PostHog instance within the Netdata infrastructure. Read about the information collected, and learn how to-opt, on our [anonymous statistics](/docs/netdata-agent/configuration/anonymous-telemetry-events.md) page.
-
-The usage statistics are _vital_ for us, as we use them to discover bugs and prioritize new features. We thank you for
-_actively_ contributing to Netdata's future.
-
-### Troubleshooting and known issues
-
-We are tracking a few issues related to installation and packaging.
-
-#### Installs on hosts without IPv4 connectivity
-
-Our regular installation process requires access to a number of GitHub services that do not have IPv6 connectivity. As
-such, using the kickstart install script on such hosts generally does not work, and will typically fail with an
-error from cURL or wget about connection timeouts. You can check if your system is affected by this by attempting
-to connect to (or ping) `https://api.github.com/`. Failing to connect indicates that you are affected by this issue.
-
-There are three potential workarounds for this:
-
-1. You can configure your system with a proper IPv6 transition mechanism, such as NAT64. GitHub’s anachronisms
-   affect many projects other than just Netdata, and there are unfortunately a number of other services out there
-   that do not provide IPv6 connectivity, so taking this route is likely to save you time in the future as well.
-2. If you are using a system that we publish native packages for (see our [platform support
-   policy](/docs/netdata-agent/versions-and-platforms.md) for more details),
-   you can manually set up our native package repositories as outlined in our [native package install
-   documentation](/packaging/installer/methods/packages.md). Our official
-   package repositories do provide service over IPv6, so they work without issue on hosts without IPv4 connectivity.
-3. If neither of the above options work for you, you can still install using our [offline installation
-   instructions](/packaging/installer/methods/offline.md), though
-   do note that the offline install source must be prepared from a system with IPv4 connectivity.
```

Comment:
This section is still accurate and is absolutely relevant to support. It still comes up on occasion with users who have not read the documentation, it’s non-obvious, and if a user runs into this they _CANNOT INSTALL NETDATA WITHOUT RESOLVING IT_.

Reply:
made a new file for this, that will be inside the linux section in our docs